### PR TITLE
console: Add go-generated bindata check test

### PR DESCRIPTION
**Pull request by hyochan-brown**
## Proposed changes
- Add go-generated bindata file check test for security
  - assets.go
  - bindata.go
- Run the go-bindata command and check the hash value of bindata file is same.
- go-bindata version is fixed to 3.2.2 to prevent the test fail when go-bindata version up.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: console/jsre/deps/bindata_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package deps
```
Comment (jeongkyun-oh) [inline console/jsre/deps/bindata_test.go:1]: We need to add a copyright

File: node/cn/tracers/internal/tracers/assets_test.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package tracers
```
Comment (jeongkyun-oh) [inline node/cn/tracers/internal/tracers/assets_test.go:1]: @hyochan-brown We need to add a copyright

File: Makefile [hunk 3]
```diff
@@ -121,3 +121,10 @@ devtools:
 	@type "npm" 2> /dev/null || echo 'Please install node.js and npm'
 	@type "solc" 2> /dev/null || echo 'Please install solc'
 	@type "protoc" 2> /dev/null || echo 'Please install protoc'
+
+# The devtoolsForTest target installs tools required for 'go-bindata'.
+# You need to put $BIN (or $GOPATH/bin) in your PATH to use 'go-bindata'.
+# If you upload bindata file, you must update under go-bindata version same wtih yours.
+
+devtools-for-test:
+	env GOFLAGS= GOBIN= go get -u github.com/kevinburke/go-bindata/go-bindata@v3.22.0
```
Comment (jeongkyun-oh) [inline Makefile:130]: If we need to install this package all the time, how about adding this package to `go.mod`?

Comment (hyochan-brown) [reply Makefile]: Thanks JK!
go-bindata run on command...
I think adding dependency in go.mod doesn't works in command..

Comment (jeongkyun-oh) [reply Makefile:130]: Oh, I think I misunderstood for the test. Sorry for confusion.

**Comment by yoomee1313** [comment]
This PR is closed. Previous work will be moved to a new branch and resumed later with a new PR. https://github.com/yoomee1313/klaytn/pull/new/feature/add-gogenerated-bindata-check-test

**Pull request closed without merging**